dipy/tracking/tissue_classifier.pxd

```diff
@@ -1,5 +1,5 @@
 
-cdef enum TissueClass:
+cpdef enum StreamlineStatus:
```

Comment:
Why the `cdef` became a `cpdef`?

Reply:
It needs to be accessed in python files. Enum could not be accessed in python file before cython 0.21. We had it redefined in tissue_classifier.py before. I removed it, and set the cpdef, following the new cython minimum requirements.

Reply:
That should not have any performance changes right? Let's make sure.

Reply:
Thanks for the comment. Tested, there is no difference in speed.